feature-sessions/src/main/java/io/github/droidkaigi/confsched2022/feature/sessions/TimetableItem.kt

```diff
@@ -47,9 +46,9 @@ fun TimetableItem(
             color = Color.White,
             modifier = Modifier
                 .padding(bottom = 8.dp)
+                .weight(weight = 1f, fill = false)
```

Comment:
I didn't know the fill = false. So let me put an awesome label on this issue. 🏷️